src/peerconnection.cpp

```diff
@@ -256,8 +256,8 @@ bool PeerConnection::hasMedia() const {
 }
 
 std::shared_ptr<Track> PeerConnection::addTrack(Description::Media description) {
-	if (localDescription())
-		throw std::logic_error("Tracks must be created before local description");
+//	if (localDescription())
```

Comment:
I did this so that I can add more tracks to an active peer connection. Am I opening a whole can of worms here?

Reply:
Yeah, renegociation is not supported so the new track won't be propagated to the remote peer if you add it after setting the local description.

Reply:
Now they are 😎.

Reply:
OK! I'm taking a note to implement renegotiation also for data channels later.